index.html

```diff
@@ -1381,7 +1467,7 @@ <h2>Verifiable Profile</h2>
               either unrestricted or extended in the future.</li>
             </ul>
       <p>The following example demonstrates how to express a
-      simple <a>verifiable profile</a>.</p>
+      simple <a>verifiable profile</a>. STILL TO DO</p>
```

Comment:
If you must put in a comment like "STILL TO DO", do so using <p class="issue">STILL TO DO</a> and explain what still needs to be done.

Reply:
Sorry!!

Reply:
@David-Chadwick I noticed that my markup was overwritten, fixed it, see example above again to see what I was talking about.